openpype/hosts/houdini/api/plugin.py

```diff
@@ -84,3 +90,109 @@ def process(self):
                 OpenPypeCreatorError,
                 OpenPypeCreatorError("Creator error: {}".format(er)),
                 sys.exc_info()[2])
+
+
+@six.add_metaclass(ABCMeta)
+class HoudiniCreator(NewCreator):
+    selected_nodes = []
+
+    @staticmethod
+    def _create_instance_node(
+            node_name, parent,
+            node_type="geometry"):
+        # type: (str, str, str) -> hou.Node
+        """Create node representing instance.
+
+        Arguments:
+            node_name (str): Name of the new node.
+            parent (str): Name of the parent node.
+            node_type (str, optional): Type of the node.
+
+        Returns:
+            hou.Node: Newly created instance node.
+
+        """
+        parent_node = hou.node(parent)
+        instance_node = parent_node.createNode(
+            node_type, node_name=node_name)
+        instance_node.moveToGoodPosition()
+        return instance_node
+
+    def create(self, subset_name, instance_data, pre_create_data):
+        try:
+            if pre_create_data.get("use_selection"):
+                self.selected_nodes = hou.selectedNodes()
```

Comment:
```suggestion
            if pre_create_data.get("use_selection"):
                self.selected_nodes = hou.selectedNodes()
            else:
                self.selected_nodes = []
```

Reply:
Also, shouldn't there be error raised - since `use selection` is ON and no selection is active?